### DISCUSSION TITLE
简化直播间 漏网元素；删除广告 视频和动态下小喇叭屏蔽失效

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#4444] [功能建议 / Ideas]
1.简化直播间 聊天框内官方的一些通知没有屏蔽选项
<div class="chat-item common-danmuku-msg border-box" style=" border-radius: 4px; padding: 8px 6px; line-height: 1.5;"><div style="color: #998EFF; display: inline; vertical-align: middle;"><span style="color: #998EFF;">恭喜主播 XXX </span>成为 上小时人气榜 第 27 名！</div></div>
<div class="chat-item common-danmuku-msg border-box" style=" border-radius: 4px; padding: 8px 6px; line-height: 1.5;"><div style="color: #F294AE; display: inline; vertical-align: middle;">2023BLS秋季赛·全站总选赛：由于房间内的用户完成发送弹幕任务，恭喜你今日累计获得额外300闪耀值加分！每日加分上限500闪耀值哦～</div></div>

2.视频页面的活动横幅下方评论区上方的黄色小喇叭通知以及动态的黄色小喇叭 这俩都失效了（没开保留小喇叭）
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/7b51fb56-1182-47e2-ba21-e2b8edf1b8c1)
——————————
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/9107dc9e-47c7-4331-bd22-f79d0be5926d)
——————————

**Comment by pinaZC** [comment]
div class="chat-item common-danmuku-msg border-box" style=" border-radius: 4px; padding: 8px 6px; line-height: 1.5;"<div style="color: #F294AE; display: inline; vertical-align: middle;">距离锦鲤小时榜结束还有5分钟，最高10倍的锦鲤红包就在前方！
div class="chat-item common-danmuku-msg border-box" style=" border-radius: 4px; padding: 8px 6px; line-height: 1.5;"<div style="color: #998EFF; display: inline; vertical-align: middle;">恭喜主播 xxx 成为单机航海第17名</div></div>
div class="chat-item common-danmuku-msg border-box" style=" border-radius: 4px; padding: 8px 6px; line-height: 1.5;"<div style="color: #998EFF; display: inline; vertical-align: middle;"><span style="color: #998EFF;">恭喜主播 xxx </span>成为 上小时人气榜 第 8 名！</div></div>
额 直接复制没发出来 这样可不可以 看上去属于同一类

**Comment by the1812** [comment]
小喇叭那个移动到简化评论区了

**Reply by Geeyun-JY3** [reply]
2.8.5（正式版和预览版）这个功能的描述是混乱的，`简化评论区`和`删除广告`各有一个相关开关：

- `简化评论区`的`活动横幅`（根据 <https://github.com/the1812/Bilibili-Evolved/commit/eb5e2252497a1a84ccf2174f7e65d8e895ebad97>，这个选项就是针对小喇叭横幅的）：
  https://github.com/the1812/Bilibili-Evolved/blob/d0625315bf9896d08381787b8bb102c2d9ed7286/registry/lib/components/style/simplify/comments/index.md?plain=1#L13
  ![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/6d5ccd05-abd1-4e57-8b41-bfdc9b6f38a9)
- `删除广告`的`保留小喇叭通知`：
  https://github.com/the1812/Bilibili-Evolved/blob/d0625315bf9896d08381787b8bb102c2d9ed7286/registry/lib/components/utils/remove-promotions/index.ts#L109
  ![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/14a6993b-a086-4b2d-9faa-d88d42af2579)

且`简化评论区`中的`活动横幅`选项的名称和描述还会与`删除广告`的`保留活动横幅`选项搞混：

- https://github.com/the1812/Bilibili-Evolved/blob/d0625315bf9896d08381787b8bb102c2d9ed7286/registry/lib/components/style/simplify/comments/index.md?plain=1#L13
- https://github.com/the1812/Bilibili-Evolved/blob/d0625315bf9896d08381787b8bb102c2d9ed7286/registry/lib/components/utils/remove-promotions/index.ts#L108

***

另外为啥要把这个删除小喇叭横幅移到`简化评论区`里去呢。
从视觉上看这个小喇叭横幅更像是一种见缝插针的广告（虽然说在个人空间动态 tab 好像确实需要点击评论按钮后才能看见这东西），位于评论区标题`评论`二字上方，和当前视频所参与活动横幅或普通的广告横幅下方，刚好隔开了视频主体相关内容和评论区，所以个人还是建议将该选项放在`删除广告`里。关联：https://github.com/the1812/Bilibili-Evolved/pull/4199#issuecomment-1555887629

**Reply by the1812** [reply]
这个是 `删除广告` 那边忘更新描述了, `简化评论区` 这边我也改成 `小喇叭横幅` 吧, 避免误会.

放在 `简化评论区` 里是因为它一定是跟随评论区一起出现的, 并且内容也不一定是广告 (有时候就是正常的站内活动), 在 #4199 之前也是一直属于 `简化评论区` 的功能, 这次对其新增了细分选项后就顺便挪回去了.

**Reply by Geeyun-JY3** [reply]
明白了。
